lib/veri_helpers.ml

```diff
@@ -0,0 +1,45 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std
+
+module Dis  = Disasm_expert.Basic
+
+type insn = Dis.full_insn
+
+let lift_insn arch (mem,insn) = 
+  let module Target = (val target_of_arch arch) in
+  match Target.lift mem insn with
+  | Ok bil -> Some bil
+  | Error _ -> None
+
+let insn_of_mem dis mem =
+  let open Or_error in
+  Dis.insn_of_mem dis mem >>= (fun (imem, insn, left) -> 
```

Comment:
if something is `left`, then it is also an error. In general. You should report this.